services/dydx_client/dydx_p_client.py

```diff
@@ -2,19 +2,16 @@
 from dydx3 import Client
 from web3 import Web3
 
-
 """
 This class DydxPClient is responsible for initializing the DydxClient instance.
 It has a function __create_dydx_Instance() that is responsible for initializing the dydx instance and returning it.
 """
-
+CLIENT = None
```

Comment:
we should create an initializer, def __init__, inside the DydxPClient and initialize the client there. Take a look at one of the classes in Hedging Scripts.